Source/SimCacheModule/Source/Module/SimCacheModule.cpp

```diff
@@ -136,6 +133,12 @@ bool SimCacheModule::InitializeSimConnectClient()
 		return false;
 	}
 
+	SimConnectClient->GetOnConnectionOpenFunc() = [ & ]()
```

Comment:
Not really an issue (especially since this is a placeholder), but just a remark: this reads a little weird to me (a function that says _get_ that you're _assigning to_?), but it makes sense since it returns it by reference?